src/style.css

```diff
@@ -1 +1,9 @@
 /* styles go here */
+
+div {
+  width: 100%;
+}
+
+span {
+  display: inline;
+}
```

Comment:
those are default styles for the corresponding elements,
if you need to add some styles, add 
```
* {
  box-sizing: border-box;
}
```